Quick improvment for config in db (#4378)
mattermost/docs: source/administration/config-in-database.rst

Another way is to view your ``config.json`` file and get the value in ``SqlSettings.DataSource``.

> If ``SqlSettings.DataSource`` does not start with ``postgres://`` or ``mysql://`` then you have to add it to the beginning depending which database is in use. Also, if you see ``\u0026`` replace it with ``&``.

Reviewer: ```suggestion
If ``SqlSettings.DataSource`` does not start with ``postgres://`` or ``mysql://``, then you have to add this line to the beginning based on the database in use. Also, if you see ``\u0026`` replace it with ``&``.
```
